From 03797e535701ecd07c1cab6c264b9ce403bf1182 Mon Sep 17 00:00:00 2001
From: Vladimir 'phcoder' Serbinenko <phcoder@gmail.com>
Date: Fri, 26 Apr 2013 11:26:08 +0200
Subject: [PATCH 353/482] 	* configure.ac: Fix loongson conditional.

---
 ChangeLog    | 4 ++++
 configure.ac | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 0bc5cc4..f3d1352 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2013-04-26  Vladimir Serbinenko  <phcoder@gmail.com>
+
+	* configure.ac: Fix loongson conditional.
+
 2013-04-25  Vladimir Serbinenko  <phcoder@gmail.com>
 
 	Enable mipsel-arc.
diff --git a/configure.ac b/configure.ac
index 4af1a03..7d3aeac 100644
--- a/configure.ac
+++ b/configure.ac
@@ -208,7 +208,7 @@ case "${target_cpu}-$platform" in
         TARGET_LINK_ADDR=0x80700000
 	TARGET_DECOMPRESSOR_LINK_ADDR=0x80600000
 	;;
-     mips*-qemu_mips | mips-loongson)
+     mips*-qemu_mips | mips*-loongson)
         TARGET_DECOMPRESSOR_LINK_ADDR=0x80100000
 	;;
 esac
-- 
1.8.2.1

