From 56b79f793db3488f8efa3bc345cd71b179c03c29 Mon Sep 17 00:00:00 2001
From: Vladimir 'phcoder' Serbinenko <phcoder@gmail.com>
Date: Tue, 7 May 2013 11:45:47 +0200
Subject: [PATCH 435/482] 	* autogen.sh: Exclude unused libgcrypt files
 from translation.

---
 ChangeLog  | 4 ++++
 autogen.sh | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 92af2dd..d6a68b7 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,9 @@
 2013-05-07  Vladimir Serbinenko  <phcoder@gmail.com>
 
+	* autogen.sh: Exclude unused libgcrypt files from translation.
+
+2013-05-07  Vladimir Serbinenko  <phcoder@gmail.com>
+
 	Simplify few strings.
 
 2013-05-07  Vladimir Serbinenko  <phcoder@gmail.com>
diff --git a/autogen.sh b/autogen.sh
index 00fdafc..60ae5db 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -6,7 +6,7 @@ export LC_CTYPE=C
 export LC_COLLATE=C
 unset LC_ALL
 
-find . -iname '*.[ch]' -not -ipath './grub-core/lib/libgcrypt-grub/*' -not -ipath './build-aux/*' |sort > po/POTFILES.in
+find . -iname '*.[ch]' -not -ipath './grub-core/lib/libgcrypt-grub/*' -not -ipath './build-aux/*' -not -ipath './grub-core/lib/libgcrypt/src/misc.c' -not -ipath './grub-core/lib/libgcrypt/src/global.c' -not -ipath './grub-core/lib/libgcrypt/src/secmem.c' |sort > po/POTFILES.in
 
 autogen --version >/dev/null || exit 1
 
-- 
1.8.2.1

