diff -urN grub-2.00.orig/util/grub.d/10_linux.in grub-2.00/util/grub.d/10_linux.in
--- grub-2.00.orig/util/grub.d/10_linux.in	2013-04-11 19:37:14.409778809 +0400
+++ grub-2.00/util/grub.d/10_linux.in	2013-04-11 19:37:24.954813737 +0400
@@ -29,7 +29,11 @@
 CLASS="--class gnu-linux --class gnu --class os"
 
 if [ "x${GRUB_DISTRIBUTOR}" = "x" ] ; then
-  OS="$(sed 's, release .*$,,g' /etc/system-release)"
+  if [ -f /etc/rfremix-release ]; then
+    OS="$(sed 's, release .*$,,g' /etc/rfremix-release)"
+  else
+    OS="$(sed 's, release .*$,,g' /etc/system-release)"
+  fi
 else
   OS="${GRUB_DISTRIBUTOR}"
   CLASS="--class $(echo ${GRUB_DISTRIBUTOR} | tr 'A-Z' 'a-z' | cut -d' ' -f1) ${CLASS}"
diff -urN grub-2.00.orig/util/grub.d/20_linux_xen.in grub-2.00/util/grub.d/20_linux_xen.in
--- grub-2.00.orig/util/grub.d/20_linux_xen.in	2013-04-11 19:37:14.409778809 +0400
+++ grub-2.00/util/grub.d/20_linux_xen.in	2013-04-11 19:37:24.955813740 +0400
@@ -29,7 +29,11 @@
 CLASS="--class gnu-linux --class gnu --class os --class xen"
 
 if [ "x${GRUB_DISTRIBUTOR}" = "x" ] ; then
-  OS="$(sed 's, release .*$,,g' /etc/system-release)"
+  if [ -f /etc/rfremix-release ]; then
+    OS="$(sed 's, release .*$,,g' /etc/rfremix-release)"
+  else
+    OS="$(sed 's, release .*$,,g' /etc/system-release)"
+  fi
 else
   OS="${GRUB_DISTRIBUTOR}"
   CLASS="--class $(echo ${GRUB_DISTRIBUTOR} | tr 'A-Z' 'a-z' | cut -d' ' -f1) ${CLASS}"
